tsMuxer/hevcStreamReader.h

```diff
@@ -41,6 +42,7 @@ class HEVCStreamReader: public MPEGStreamReader
     HevcVpsUnit* m_vps;
     HevcSpsUnit* m_sps;
     HevcPpsUnit* m_pps;
+	HevcSeiUnit* m_sei;

```

Comment:
Any reason why it's not a `std::unique_ptr`?

Reply:
@xavery not sure what you mean here. I used the same struct for sei as for vps, sps and pps.

Reply:
There is no reason for this to be a raw pointer : since C++11, we can use the `std::unique_ptr` class, which guarantees that the pointer is never "accidentally" copied and its ownership is always tied to the `unique_ptr` object. Existing code doesn't do this, because it was written before these classes were available.